### ISSUE TITLE
Add adc_mV calibration support to LIS3DH blocks

### ISSUE THREAD

**Issue by zfi** [parallaxinc/solo#302]
The field reports note that there is a significant variance in the initial offset value for adc output among a group of devices. Add support to the LIS3DH initialization block to collect an offset value between -500mv and 500mv. Call the lis3dh_adcCal_mV(LIS3DH_1, __offset__) function to provide the calibration value to the library code.

**Comment by zfi** [comment]
Prototype blocks look like this:
<img width="478" alt="Screen Shot 2020-02-24 at 4 25 17 PM" src="https://user-images.githubusercontent.com/8442260/75203213-64363480-5722-11ea-9ee4-67889f8e36e1.png">

When combined with other initialization options, the ADC voltage offset remains at the bottom of the init block.
<img width="487" alt="Screen Shot 2020-02-24 at 4 27 45 PM" src="https://user-images.githubusercontent.com/8442260/75203285-9cd60e00-5722-11ea-97c6-8a6f7029fc94.png">

Generated code:
```
// ------ Libraries and Definitions ------
#include "simpletools.h"
#include "lis3dh.h"

// ------ Global Variables and Objects ------
lis3dh *lis3dh_sensor;
int var_ad1;
int var_ad2;
int var_ad3;
int item;



// ------ Main Program ------
int main() {
  lis3dh_sensor = lis3dh_init(5, 6, 7);
  lis3dh_adcCal_mV(LIS3DH_1, -123);

  lis3dh_adc_mV(lis3dh_sensor, &var_ad1, &var_ad2, &var_ad3);
  lis3dh_tilt(lis3dh_sensor, &item, &item, &item, &item);
}
```

**Comment by zfi** [comment]
Released in Solo 1.4.2
